frontend/src/app/app-events.service.ts

```diff
@@ -19,12 +19,13 @@ export interface AppEvent {
   providedIn: 'root'
 })
 export class AppEventsService implements OnDestroy {
+  BUFFERSIZE: number = 4;
```

Comment:
Given the speed of `MobileNetV1` detector, let's bump this to ~10 for the time being.